rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -183,6 +183,13 @@ def _save_module_metadata(self, *args, **kwargs):
     def _module_metadata(self, *args, **kwargs):
         return self.unwrapped()._module_metadata(*args, **kwargs)
 
+    # TODO (sven): Figure out a better way to avoid having to method-spam this wrapper
```

Comment:
We need to find a better solution for this, but this fixes PPO + multi-GPU on the new stack for now.